ci-runner/src/main.rs

```diff
@@ -626,7 +629,7 @@ async fn run_remote(config_path: String) {
 
         match poll {
             Ok(res) => {
-                let mut client = match RemoteServerRunner::new("ci.butactuallyin.space:9876", sender, res).await {
+                let mut client = match RemoteServerRunner::new(&runner_config.server_address, sender, res).await {
```

Comment:
this requires `server_address` to include `:9876`. that's fine, but overall `server_address` isn't super obvious that it's the machine API address rather than the human pages. might be better to accept an `api_host` here, with an optional-but-default-of-9876 port that's tacked on?